### ISSUE TITLE
Accessibility Error: Missing form label and Alerts: Orphaned form label

### ISSUE THREAD

**Issue by elisabeth-boelter** [SergeyMosin/Appointments#650]
Using the "WAVE web accessibility evaluation tool" https://wave.webaim.org/

to test my appointment Site (https://cloud.zukunft-gestalten.online/index.php/apps/appointments/pub/A_sj%2BnAltTESPLQ%3D/form)

I get the error message: Missing form label for the field Date & Time.

2 Alerts: Orphaned form label
For the Headline Date & Time and the fuction to select the timezone.

1 Alert: Noscript element is displayed in the top left corner.

Do you have a tip, how I can solve this.

Many thanks in advance.

Elisabeth
